### ISSUE TITLE
SysMon not sending any data

### ISSUE THREAD

**Issue by rcav8r** [ukncsc/lme#18]
Hi;
First off, thanks for a great project.  Directions were easy to follow.  One type-o I did note, in chapter one it says to use use TCP port 5986 on the diagram of how the client talks to the server, and the rest of the document it says to allow 5985.

On to the question/issue.  I can not get the client to send data.  The SysMon64 services is running. (tried re-starting with no change)  I modified the client GPO to point to my server.  Had the FQDN in there, and also tried the IP address.
On my client, I see events populating in eventviewer/Applications and Services Log/Microsoft/Windows/Sysmon/Operational .
On the server the LME subscription shows as active with 0 clients connected.

I ran wireshark on my PC and do not see it even trying to send anything to the server, so I am guessing the issue is on the client side.    In Chapter 1 where it mentions IP filtering, I had that set, but removed it for testing.  No change.   Also added firewall ports to be allowed, and then turned off firewall for testing.  Still no change.    I have done a gpupdate /force when I made a change to the GPO. 

Thanks
Dave

**Comment by duncan-ncc** [comment]
Hi Dave, 

Thanks for that, I will update the diagram too have the correct port (one is HTTP vs HTTPS but both are encrypted regardless)
Have you confirmed using rsop or gpresult (or similar) that the client has received the event forwarding policy? 
It might also be worth checking the following event log to see if there are any errors or messages which suggest issues. 
```eventviewer/Applications and Services Log/Microsoft/Windows/Eventlog-ForwardingPlugin/Operational```

**Comment by rcav8r** [comment]
Thanks for the quick reply.   I got it working by changing from IP to FQDN in the client GPO.  As soon as I did that it immediately showed up as a subscription in the Windows Event Collector.  I had the FQDN there earlier, but it wasn't working so I tried the IP. I'm sure something I changed in the earlier was the real fix.    At this point I would assume that the FQDN is needed, and not the IP.
THANKS again.
